circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
+  services:
+    - docker
+
+test:
```

Comment:
We discussed this during our DevOps sync. Testing containers in Circle needs to be a part of a ticket/project. For now, a comment that describes why we're doing what we're doing will suffice.